### PR TITLE
UGENE-6170. Correct names of document formats in two workflow elements

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: src/corelibs/U2Designer/src/DelegateEditors.h [hunk 1]
```diff
@@ -201,13 +202,31 @@ private slots:
     QVariantMap spinProperties;
 };
 
-class U2DESIGNER_EXPORT ComboBoxDelegate : public PropertyDelegate {
+/** Base class for all combo-box delegates. Includes item name formatter. */
+class U2DESIGNER_EXPORT ComboBoxBaseDelegate : public PropertyDelegate {
+    Q_OBJECT
+public:
+    ComboBoxBaseDelegate(QObject *parent = 0);
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2Designer/src/DelegateEditors.h]: `nullptr` is a bit better

**Review by DmitriiSukhomlinov — DISMISSED** [review]

**Review by smlnkcv — COMMENTED** [review]
1.  "Filter BAM/SAM files": "Output format" attribute visible value in the same way: format names should be used instead if format IDs. Now  they are "sam", "bam"

File: src/corelibs/U2Designer/src/DelegateEditors.h (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2Designer/src/DelegateEditors.h]: Agree. Fixed.

**Comment by yalgaer** [comment]
>     1. "Filter BAM/SAM files": "Output format" attribute visible value in the same way: format names should be used instead if format IDs. Now  they are "sam", "bam"

Fixed

**Pull request merged**